.github/workflows/saucelabs.yaml

```
@@ -0,0 +1,40 @@
+name: Integration tests (via Sauce Labs)
+
+# Our Sauce Labs account only allows 5 tests to run at a time
+concurrency: saucelabs
+
+on:
+  pull_request:
```

Somewhat personal preference, but I find the 'key with nothing after it' slightly strange, and prefer to use [the list style](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#example-using-a-list-of-events) unless one of the events needs config:

```suggestion
on: [pull_request]
```